src/routes/categories/components/CategoryListItem.js

```diff
@@ -4,11 +4,58 @@ import * as React from 'react'
 
 import CategoryModel from 'modules/endpoint/models/CategoryModel'
 import iconPlaceholder from '../assets/IconPlaceholder.svg'
-import {
-  CategoryCaption,
-  CategoryThumbnail, Row,
-  StyledLink, SubCategoryCaption, SubCategory
-} from './CategoryListItem.styles'
```

Comment:
Why did you move these out of the .styles folder?

Reply:
I think we agreed on doing this, see https://issues.integreat-app.de/browse/WEBAPP-314

Reply:
Hmm never seen this task before :D

But it should also be made within this task then  🙃

Reply:
Hmm I don't see a problem to do this here, makes the other task way smaller.

Reply:
Not that you need to revert it, but we shouldn't randomly start doing parts of a some different task :)
Also I'm not yet convinced, that we should even do it ^^